ompi/mca/coll/adapt/coll_adapt_ireduce.c

```diff
@@ -61,7 +61,7 @@ int ompi_coll_adapt_ireduce_register(void)
 
     mca_coll_adapt_component.adapt_ireduce_max_recv_requests = 3;
     mca_base_component_var_register(c, "reduce_max_recv_requests",
-                                    "Maximum number of receive requests",
+                                    "Maximum number of receive requests per peer",
```

Comment:
It depends on the definition of peer. We can be more precise by stating “(parent and child in the underlying collective topology)”.